unit_test/test_crypt/test_crypt.c

```diff
@@ -120,10 +120,13 @@ bool libspdm_cryptest_main(void)
     #endif /* (LIBSPDM_EDDSA_ED25519_SUPPORT) || (LIBSPDM_EDDSA_ED448_SUPPORT) */
 
     #if LIBSPDM_SM2_DSA_SUPPORT
-    status = libspdm_validate_crypt_sm2();
-    if (!status) {
-        return status;
-    }
+
+    /* the Crypto SM2 key Exchange is not support now
+     * status = libspdm_validate_crypt_sm2();
+     * if (!status) {
+     *     return status;
+     * }
+     **/
```

Comment:
If we can add crypto configuration for test, then we dont need comment or delete.
We can just disable it by macro.